arangod/Transaction/Methods.cpp

```diff
@@ -2734,8 +2708,7 @@ futures::Future<OperationResult> transaction::Methods::allLocal(
   resultBuilder.openArray();
 
   auto iterator =
-      indexScan(monitor, collectionName, transaction::Methods::CursorType::ALL,
-                ReadOwnWrites::no);
+      co_await indexScan(collectionName, CursorType::ALL, ReadOwnWrites::no);
```

Comment:
It seems that the local `monitor` variable (line 2706) can now be removed as well.